templates/vite.config.txt

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'vite'
+import adonis from '@adonisjs/vite-plugin-adonis'
```

Comment:
This package is not yet published. I will create a repo in the org right now, I already have a first basic version

Reply:
Cannot we use the community package already built for Vite? https://github.com/91bytes/adonis-vite

Reply:
I took a look at it but the package lacks many features that we would need. Then I would like to have full control over the plugin because it will really be the core of our integration with Vite

( If ever the author passes by, I don't denigrate your work at all, it was a very cool initiative and a good start 💖 )

Reply:
Yeah, I will also be in the favor of writing our own plugin from scratch and have full control over it. Atleast, that is what I do all the time :)